doc/examples/eightQueens.x

```diff
@@ -0,0 +1,133 @@
+/**
+ * A solver for the classic 8-queens problem.
+ *
+ * @see https://rosettacode.org/wiki/N-queens_problem
+ */
+module eightQueens {
+    void run() {
```

Comment:
*nitpick* I am not 100% sure, but given the general look of XTC code, there is probably a naming convention for XTC Modules similar to the one in Java? Pretty much all modules in our test and example code are named in CapitalCamelCase, and not lowerCamelCase. That means that this should be named EightQueens for consistency.

Reply:
Code directly from @cpurdy , leaving it for him to rename.